runtime/doc/msgpack_rpc.txt

```diff
@@ -413,5 +417,9 @@ of update.
 ["popupmenu_hide"]
 	The popupmenu is hidden.
 
+["tabline_update", curtab, tabs]
+	Nvim will send this event when drawing tabline. curtab is the tab id
+	of the current tab. tabs is an arrays of the form [tabid, tabname].
```

Comment:
tabs is an arrays of the form [tabid, { "name": name } ].

Then we can add new fields later.